autopush/web/registration.py

```diff
@@ -68,8 +68,11 @@ def convert_chid(self, data):
 
 
 class TokenSchema(SubInfoSchema):
-    """Validates that token and optionally key/channelID"""
+    """Filters allowed values from body data"""
     token = fields.Str(allow_none=True)
+    # TODO: Temporarily allow 'aps' definition data for iOS.
```

Comment:
Yes, exactly. We don't know the values that iOS will require, thus we can't lock them down right now. We have to allow everything. When iOS completes and we understand what we need to allow, then the `TODO` would be addressed.